cluster-stamp.bicep

```diff
@@ -957,7 +957,7 @@ resource paEnforceHttpsIngress 'Microsoft.Authorization/policyAssignments@2021-0
     policyDefinitionId: pdEnforceHttpsIngressId
     parameters: {
       excludedNamespaces: {
-        value: [] // None
```

Comment:
is the `value` field required to be declarative in here to work properly? If not, any difference between this and empty `dependson`? Not asking for change trying to understand the rationale/thought-process.

Reply:
yeah, it's important here to have `[]` in this case.  This is two fold.

1) This helps be clear that we are NOT excluding any namespaces (including `kube-system`).  The intent here is to show that we want this rule cluster-wide.  Practically speaking, `gatekeeper-system` and `kube-system` are hardcoded in the `Deployment` as being skipped anyway for gatekeeper, but the default policy position is to list them explicitly anyway just in case that changes on the AKS-managed `Deployment` -- but in this case, this policy is safe to apply to all the places, so making it clear (intentional).

2) and most importantly, the policy definition's default is the common trio of namespaces, so if we don't include this `excludedNamespaces` it comes in as `null` and then gets the common trio added, which is not what we want in this case.

Reply:
Sounds good. Thanks.